content/lxd/getting-started-cli.md

```diff
@@ -42,7 +42,7 @@
 ## Choose your release
 LXD upstream maintains three release branches in parallel:
 
- * LTS release (LXD 4.0.x or LXD 3.0.x)
+ * LTS releases (LXD 4.0.x or LXD 3.0.x)
```

Comment:
Here would be good to add a short definition of LTS compared to Feature releases.

Reply:
Yeah, something like `Long term support (LTS) releases (LXD 4.0.x or LXD 3.0.x)`, should do. Then we can use the acronym in the rest of the page.

Reply:
Ok, I will add @stgraber's version.